docs/Analyzer Configuration.md

```diff
@@ -874,3 +874,16 @@ If set to true, all IEnumerable type parameters would be assumed enumerated by t
 Default Value: `false`
 
 Example: `dotnet_code_quality.CA1851.assume_method_enumerates_parameters = true`
+
+### Skip analysis if InternalsVisibleTo is present
```

Comment:
Please file an issue on https://github.com/dotnet/docs to track updating the documentation for the rules to mention this option.

Reply:
https://github.com/dotnet/docs/issues/35116